src/content/developers/docs/programming-languages/dart/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ethereum for Dart Developers
+description: Learn how to develop for Ethereum using the Dart language
+lang: en
+sidebar: true
+incomplete: true
+---
+
+## Getting Started with Smart Contracts and the Solidity Language {#getting-started-with-smart-contracts-and-solidity}
+
+## Tutorials {#tutorials}
+
+1. [This tutorial](https://www.geeksforgeeks.org/flutter-and-blockchain-hello-world-dapp/) takes you through all the steps to get started:
+   1. Installing the [Truffle development suite](https://www.trufflesuite.com/)
+   2. Writing a smart contract in [Solidity](https://soliditylang.org/)
+   3. Writing a user interface in Dart
+2. [This tutorial](https://medium.com/dash-community/building-a-mobile-dapp-with-flutter-be945c80315a) is a lot shorter, which might be better
+   if you already know the basics.
+4. If you prefer the learn by watching a video, [watch this tutorial](https://www.youtube.com/watch?v=3Eeh3pJ6PeA), which is about an hour long
+5. If you are impatient, you might prefer [this tutorial](https://www.youtube.com/watch?v=jaMFEOCq_1s), which is only about twenty minutes
+
+## Working with Ethereum clients {#working-with-ethereum-clients}
+
+You can use Ethereum to create decentralized applications (or "dapps") that utilize the benefits of cryptocurrency and blockchain technology.
+There are at least two currently maintained libraries for Dart to use the JSON RPC API for Ethereum.
+
+1. [Web3dart from simonbutler.eu](https://pub.dev/packages/web3dart)
+1. [Ethereum 5.0.0 from darticulate.com](https://pub.dev/packages/ethereum)
+
+As well as additional libraries that allow you manipulate specifically Ethereum addresses, or that let you retrieve prices of various crypto
```

Comment:
"There are also additional libraries that allow you to manipulate specific Ethereum addresses, or that let you retrieve prices of various cryptocurrencies."